src/ext/java/org/opentripplanner/ext/transmodelapi/model/siri/et/EstimatedCallType.java

```diff
@@ -121,16 +120,29 @@ public static GraphQLObjectType create(
                                     TripTimeOnDate tripTimeOnDate = environment.getSource();
                                     if (tripTimeOnDate.getActualDeparture() == -1) { return null; }
                                     return 1000 * (
-                                        tripTimeOnDate.getServiceDay() +
-                                        tripTimeOnDate.getActualDeparture());
-                              })
-                           .build())
+                                            tripTimeOnDate.getServiceDay() +
+                                                    tripTimeOnDate.getActualDeparture()
+                                    );
+                                })
+                    .build())
             .field(GraphQLFieldDefinition.newFieldDefinition()
                     .name("timingPoint")
                     .type(new GraphQLNonNull(Scalars.GraphQLBoolean))
-                    .description("Whether this is a timing point or not. Boarding and alighting is not allowed at timing points.")
-                    .dataFetcher(environment -> ((TripTimeOnDate) environment.getSource()).isTimepoint())
+                    .description(
+                            "Whether this is a timing point or not. Boarding and alighting is not allowed at timing points.")
```

Comment:
In GTFS, you can board and alight at timepoints. Furthermore, in the current GTFS spec timepoints are always stops (i.e., `timepoint` is an attribute of a record in stops.txt).

Reply:
We have talked about this PR internally, I will open an new and put a link here. The business logic for the NeTEx waitPoint is not OK and @hannesj  you are right, the attribute exists already, the name should be different because it is not transmodel, but I will keep it as is. @barbeau I will explain in detail the NeTEx definition of waitPoint in the next PR. It will be much cleaner and simpler.

Reply:
> 
The documentation is old, I'll try to update it to a more convinient and up to date.

Reply:
I’d suggest to just reuse this PR and force-push a new more clean commit. That way we can have the discussion and history in one place

Reply:
@barbeau @hannesj I have updated this PR and simplified it much. Please look at my notes in the description if you have any opinion. Specially the description of *timepoint* in GraphQl actually seems wrong to me.